c10/util/Backtrace.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstddef>
+#include <string>
+#include <typeinfo>
+
+#include "c10/macros/Macros.h"
+
+namespace c10 {
+C10_API std::string get_backtrace(
```

Comment:
I thought you recently changed some sites in caffe2 to `CAFFE2_API`. As a general plan, do we want to keep separate macros for c10/c2/aten or just use `C10_API` everywhere?